### PR TITLE
Initialize the CLI landing page with "Hugo"

### PR DESCRIPTION
**Pull request by gdarchen**
# Fixes issue [#cli-24](https://github.com/reacli/cli/issues/24)

*Follows #2*

**Summary:**

Initiate a landing page serving as documentation for the users. It could later embed SVGs showing how we can use the CLI.

The landing page is available here: [https://reacli.github.io](https://reacli.github.io)

**Changes:**

I added the the sources for the landing page, using the "learn" `hugo` theme.

Here is the initialization of the landing page. Its content needs to be written.

### CODE REVIEW

File: content/usage/usage.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+title: "Usage"
+date: 2019-02-18T19:57:39+01:00
+weight: 1
+chapter: true
+---
+
+#### If **Reacli** is installed globally
+
+If you have installed **Reacli** globally (see [installation]({{< relref "install.md" >}})), you can directly use **Reacli** like:
```
Comment (tanohzana) [inline content/usage/usage.md]: I would have said: "Once you have **Reacli** installed globally, you can use it like this:"

**Review by tanohzana — CHANGES_REQUESTED** [review]
Only one thing that bothers me, but the rest is great. Good job 😄

File: content/usage/usage.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ chapter: true
 
 #### If **Reacli** is installed globally
 
-If you have installed **Reacli** globally (see [installation]({{< relref "install.md" >}})), you can directly use **Reacli** like:
+If you have installed **Reacli** globally (see [installation]({{< relref "install.md" >}})), you can directly use it like this:
```
Comment (tanohzana) [inline content/usage/usage.md]: My point was than "If you have installed **Reacli** globally" is not proper English...

**Review by tanohzana — CHANGES_REQUESTED** [review]

File: content/usage/usage.md (hunk 2 above)
Comment (gdarchen) [reply content/usage/usage.md]: Sorry I completely missed it 😞

**Comment by gdarchen** [comment]
It should FINALLY be fixed now 😁 [`b79ec58`](/reacli/cli-docs/pull/3/commits/b79ec58994a8008790f1f1c6b1b472a40e2d55c7)

**Review by tanohzana — APPROVED** [review]

File: content/usage/usage.md (hunk 2 above)
Comment (tanohzana) [reply content/usage/usage.md]: b79ec58

File: content/usage/usage.md (hunk 1 above)
Comment (tanohzana) [reply content/usage/usage.md]: b79ec58

**Comment by tanohzana** [comment]
Perfect. Think about adding the commit number to each discussion like I did, so that I can check the changes and close the discussion 😉

**Comment by gdarchen** [comment]
I updated my comment and will remember your advice for my future changes on requests 😊

**Pull request merged**